notes/0.13.13/extras.md

```diff
@@ -0,0 +1,43 @@
+
+### Synthetic subprojects
+
+sbt 0.13.13 adds support for `AutoPlugin`s to generate
+synthetic subprojects. To generate subprojects, override `buildExtras`
+method as follows:
+
+    import sbt._
+    import Keys._
+
+    object BuildExtrasPlugin extends AutoPlugin {
+      override def buildExtras: Seq[Project] =
+        List("foo", "bar", "baz") map generateProject
+
+      def generateProject(id: String): Project =
+        Project(id, file(id)).
+          settings(
+            name := id
+          )
+    }
+
+In addition, subprojects may be derived from an existing subproject
+by overriding `projectExtras`.
+
+    import sbt._
+    import Keys._
+
+    object ProjectExtrasPlugin extends AutoPlugin {
+      // Enable this plugin by default
+      override def requires: Plugins = sbt.plugins.CorePlugin
+      override def trigger = allRequirements
+
+      override def projectExtras(proj: ProjectDefinition[_]): Seq[Project] =
+        // Make sure to exclude project extras to avoid recursive generation
```

Comment:
Are we not managing this sbt side because there are edge-cases where it's convenient to have (some level of) recursion? Eg. one autoplugin's projectExtras consuming a previous autoplugin's projectExtras?

If not, then can we avoid making this something the user needs to deal with?